docs/quick_start/starting.rst

```diff
@@ -18,13 +18,15 @@ For example, to start the Fledge system, open a session to the Fledge device and
 
 If authentication is enabled, which is the default mode for Fledge version 3.0 onward, then a number of the  commands require authentication. Authentication can be accomplished by several means;
 
-  - Set the environment variable *USERNAME* to be the user name.
+  - Set the environment variable *FLEDGE_USER* to be the user name.
     
   - Pass the *-u* flag flag to the command to specify a user name.
 
+  - Create an authentication file
```

Comment:
Why do we need to create it explicitly in order to bypass the environment variables?

Reply:
Because it is more secure. Every application gets passed the environment and will see the username and password set there.